app/src/main/res/layout/item_artist.xml

```diff
@@ -1,26 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="@dimen/artist_image"
+    android:layout_width="@dimen/dimen_item_actor_image"
     android:layout_height="wrap_content"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:orientation="vertical">
 
     <com.google.android.material.imageview.ShapeableImageView
         android:id="@+id/img_artist"
-        android:layout_width="@dimen/artist_image"
-        android:layout_height="@dimen/artist_image"
+        android:layout_width="@dimen/dimen_item_actor_image"
+        android:layout_height="@dimen/dimen_item_actor_image"
         android:scaleType="centerCrop"
-        android:src="@drawable/mark_ruffalo"
+        tools:src="@drawable/mark_ruffalo"
```

Comment:
If you use tools:srcCompat="@tools:sample/avatars" - you can remove mark_ruffalo.webp